dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/RelationshipRestRepository.java

```diff
@@ -291,12 +295,34 @@ private boolean isAllowedToModifyRelationship(Context context, Relationship rela
 
     @Override
     protected void delete(Context context, Integer id) throws AuthorizeException {
+        String copyVirtual =
+            requestService.getCurrentRequest().getServletRequest().getParameter("copyVirtualMetadata");
+        if (copyVirtual == null) {
+            copyVirtual = "none";
+        }
+
         Relationship relationship = null;
         try {
             relationship = relationshipService.find(context, id);
             if (relationship != null) {
                 try {
-                    relationshipService.delete(context, relationship);
+                    switch (copyVirtual) {
+                        case "all":
+                            relationshipService.delete(context, relationship, true, true);
+                            break;
+                        case "left":
+                            relationshipService.delete(context, relationship, true, false);
+                            break;
+                        case "right":
+                            relationshipService.delete(context, relationship, false, true);
+                            break;
+                        case "configured":
```

Comment:
Should each of these `case` statements reference constants like in `ItemRestRepository`?  It might be good to set this up in the same way as `ItemRestRepository` to more easily see the options available to pass to this endpoint.

Reply:
This has been adjusted